runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -848,3 +862,92 @@ TR_J9SharedCache::getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCache(TR
    uintptrj_t classChainOffsetInSharedCache = offsetInSharedCacheFromPointer(classChainIdentifyingLoaderForClazz);
    return classChainOffsetInSharedCache;
    }
+
+const void *
+TR_J9SharedCache::storeSharedData(J9VMThread *vmThread, char *key, J9SharedDataDescriptor *descriptor)
+   {
+   return _sharedCacheConfig->storeSharedData(
```

Comment:
Everything that mentions SCC data structures defined by VM needs to be protected by `#if defined(J9VM_OPT_SHARED_CLASSES)` because if the VM is configured without SCC our code won't compile